### PR TITLE
docs(readme): add CRUD to the top section of the get-started

### PR DESCRIPTION
**Pull request by hanxiao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,10 +86,102 @@ jina hello-world --help
...
+
+First we look at basic CRUD operations. In Jina, CRUD corresponds to four functions: `index` (create), `search` (read), `update`, and `delete`. With documents below as example:
+```python
+import numpy as np
+from jina import Document
+docs = [
+    Document(id='🐲', embedding=np.array([0, 0]), tags={'guardian': 'Azure Dragon', 'position': 'East'}),
+    Document(id='🐦', embedding=np.array([1, 0]), tags={'guardian': 'Vermilion Bird', 'position': 'South'}),
+    Document(id='🐢', embedding=np.array([0, 1]), tags={'guardian': 'Black Tortoise', 'position': 'North'}),
+    Document(id='🐯', embedding=np.array([1, 1]), tags={'guardian': 'White Tiger', 'position': 'West'})]
```
Comment (cristianmtr) [inline README.md:104]:
```suggestion
    Document(id='🐯', embedding=np.array([1, 1]), tags={'guardian': 'White Tiger', 'position': 'West'})
]
```

File: README.md [hunk 2]
```diff
@@ -86,10 +86,102 @@ jina hello-world --help
 
 |     |   |
 | --- |---|
-| 🐣  | [Create](#create) • [Visualize](#visualize) • [Feed Data](#feed-data) • [Fetch Result](#fetch-result) • [Construct Document](#construct-document) • [Add Logic](#add-logic) • [Inter & Intra Parallelism](#inter--intra-parallelism) • [Decentralize](#decentralized-flow) • [Asynchronous](#asynchronous-flow) |
-| 🚀  | [Customize Encoder](#customize-encoder) • [Test Encoder](#test-encoder-in-flow) • [Parallelism & Batching](#parallelism--batching) • [Add Data Indexer](#add-data-indexer) • [Compose Flow from YAML](#compose-flow-from-yaml) • [Search](#search) • [Evaluation](#evaluation) • [REST Interface](#rest-interface) |
+| 🥚  | [CRUD Functions](#CRUD-ops) |
```
Comment (cristianmtr) [inline README.md:89]:
```suggestion
| 🥚  | [CRUD Functions](#CRUD-Functions) |
```

File: README.md [hunk 3]
```diff
@@ -364,8 +456,9 @@ That's all you need to know for understanding the magic behind `hello-world`. No
 
 |     |   |
 | --- |---|
-| 🐣   | [Create](#create) • [Visualize](#visualize) • [Feed Data](#feed-data) • [Fetch Result](#fetch-result) • [Construct Document](#construct-document) • [Add Logic](#add-logic) • [Inter & Intra Parallelism](#inter--intra-parallelism) • [Decentralize](#decentralized-flow) • [Asynchronous](#asynchronous-flow) |
-| 🚀   | [Customize Encoder](#customize-encoder) • [Test Encoder](#test-encoder-in-flow) • [Parallelism & Batching](#parallelism--batching) • [Add Data Indexer](#add-data-indexer) • [Compose Flow from YAML](#compose-flow-from-yaml) • [Search](#search) • [Evaluation](#evaluation) • [REST Interface](#rest-interface) |
+| 🥚  | [CRUD Functions](#CRUD-ops) |
```
Comment (cristianmtr) [inline README.md:459]:
```suggestion
| 🥚  | [CRUD Functions](#CRUD-Functions) |
```

**Review by cristianmtr — CHANGES_REQUESTED** [review]
some minor stuff

File: README.md (hunk 1 above)
Comment (hanxiao) [reply README.md:104]: no, save line

**Pull request merged**